sceptre/cli/helpers.py

```diff
@@ -93,6 +95,13 @@ def write(var, output_format="json", no_colour=True):
 
     click.echo(output)
```

Comment:
Echo has a `file` argument that you can use to write the it to a file. https://click.palletsprojects.com/en/8.1.x/api/#click.echo. It defaults to stdout. So you could basically do:
* If `file_path` is passed, make the `file` argument a file object at that path
* If it is not passed, make the `file` argument `sys.stdout`

Reply:
Ok, I see the improvement in that, done.